### DISCUSSION TITLE
CreditCardInput won't show on mobile devices

### DISCUSSION THREAD

**Discussion by sfrunza** [weareseeed/react-square-web-payments-sdk#26] [Help]
Hi, 
I'm currently using  **CreditCardInput** with **SquarePaymentsForm** and everything works just fine on mac but the **CreditCardInput** is not showing up on mobil. I also added apple pay and google pay and still doesn't work on mobile. That's in **React 17.0.2**
Any thoughts or suggestions would be appreciated
Thanks.

**Comment by danestves** [comment]
Hi @sfrunza can you make a reproduction here 👉 https://codesandbox.io/?

**Reply by sfrunza** [reply]
Sure [https://codesandbox.io/s/jovial-panna-n4u8z?file=/src/PaymentForm.js](https://codesandbox.io/s/jovial-panna-n4u8z?file=/src/PaymentForm.js)

**Reply by sfrunza** [reply]
I open the link from my phone and don't see the card input form

**Reply by danestves** [reply]
![image](https://user-images.githubusercontent.com/31737273/148448034-77d4f1e8-3df1-4432-83e4-7385eaa6c9b6.png)

I can see the Credit Card 👀
